lisa/nic.py

```diff
@@ -132,6 +132,23 @@ class Nics(InitializableMixin):
 
     _file_not_exist = re.compile(r"No such file or directory", re.MULTILINE)
 
+    # ConnectX-3 uses mlx4_core
+    # mlx4_en and mlx4_ib depends on mlx4_core
+    #  need remove mlx4_en and mlx4_ib firstly
+    #  otherwise will see modules is in used issue
+    # ConnectX-4/ConnectX-5 uses mlx5_core
+    # mlx5_ib depends on mlx5_core, need remove mlx5_ib firstly
+    _reload_modules_dict: Dict[str, List[str]] = {
```

Comment:
Merge the two dict into one, so it's easier to manage data on same concept. You can create a class to hold multiple data on the same name.